libdnf/sack/query.cpp

```diff
@@ -1322,7 +1322,9 @@ Query::Impl::filterAdvisory(const Filter & f, Map *m, int keyname)
                     eq = false;
             }
             if (eq) {
-                pkgs = advisory.getPackages(false);
+                auto advisoryPkgs = advisory.getPackages(false);
+                pkgs.insert(pkgs.end(), std::make_move_iterator(advisoryPkgs.begin()),
+                            std::make_move_iterator(advisoryPkgs.end()));
```

Comment:
Still not understand why you have to change it.

Reply:
Output paramenters are generally bad practice. You need to go through process of Prepare - Call - Handle that is more complex than it has to be. This case is special and rare, you shouldn't force user to go through this complex process just because of this one case.